inference-engine/src/transformations/include/transformations/rt_info/disable_constant_folding.hpp

```diff
@@ -25,15 +27,19 @@ class TRANSFORMATIONS_API DisableConstantFolding {
 };
 
 TRANSFORMATIONS_API void disable_constant_folding(const std::shared_ptr<Node>& node);
-}  // namespace ngraph
 
-namespace ov {
-extern template class TRANSFORMATIONS_API VariantImpl<ngraph::DisableConstantFolding>;
+TRANSFORMATIONS_API void enable_constant_folding(const std::shared_ptr<Node>& node);
+
+TRANSFORMATIONS_API bool constant_folding_is_disabled(const std::shared_ptr<Node>& node);
+
+extern template class TRANSFORMATIONS_API VariantImpl<DisableConstantFolding>;
 
 template<>
-class TRANSFORMATIONS_API VariantWrapper<ngraph::DisableConstantFolding> : public VariantImpl<ngraph::DisableConstantFolding> {
+class TRANSFORMATIONS_API VariantWrapper<DisableConstantFolding> : public VariantImpl<DisableConstantFolding> {
 public:
-    OPENVINO_RTTI("DISABLED_CONSTANT_FOLDING");
+    OPENVINO_RTTI("disabled_constant_folding", "0");
```

Comment:
Can we use `VariantWrapper<DisableConstantFolding>`?

Reply:
`disabled_constant_folding` is currently used different components and `VariantWrapper<DisableConstantFolding>` will only add more confusion. I'm actually don't understand why should we use this long names. It doesn't make them unique. Moreover each attribute can be potentially serialized, so I would stick to short simple names to avoid renaming in future.